### ISSUE TITLE
Option to stop the game if only AIs are left

### ISSUE THREAD

**Issue by einsteinsfool** [octachrome/treason#20]
Player should be able to stop the game when only AIs are left.
Preferably AIs would stop when all players click "Stop the game" button. Because some might want to see how the game ends and they have the right to.

**Comment by octachrome** [comment]
Not sure I understand. If you want to quit the game, you can just click "leave game". What's the difference between that and stopping?

**Comment by einsteinsfool** [comment]
You're correct. When playing with only AIs and losing I can just leave and create a new room.
But from UX perspective it would be nice to have a button to just start a new game (or stop the old one) without the need to leave the room. It's more intuitive and comfortable.

Not saying this is the most important feature to implement.

**Comment by octachrome** [comment]
OK. So maybe when a player clicks "stop game" a message appears saying "1/5 players have voted to end the game". Maybe the button changes to say "you have voted to stop the game (cancel)" so you can unvote? When all humans vote to stop then the game stops.

Or how about instead of a "stop game" button we have a "fast forward" button. Clicking this sets the AI move delay to zero for the rest of the game, so it just plays out to the finish in no time at all. That way you get to see who won and who had what cards etc. Probably wouldn't need a voting system for this - any player could click it as soon as all human players are dead.

**Comment by einsteinsfool** [comment]
Good idea with "fast forward". Or maybe even just make it like that without voting? So AIs will ignore the delay after all human players are dead? Or would you prefer if it was triggered by one of the players?

**Comment by octachrome** [comment]
I think it's better with a button. I originally added the delay because people found it confusing when AIs quickly took lots of actions in sequence. Better to keep things natural and slow until a human says they want to speed things up.
